Components/Robot.py

```
@@ -1,12 +1,12 @@
-import Endstop
-import Motor
-import Ncoder
-import PaintSprayer
-import Platform
-import ScrewMotor
-import Ultrasonic_sensor
+from Components.constans import M1A,M1B,M2A,M2B,ECHO_PIN,TRIG_PIN,ENDSTOP1_PIN,ENDSTOP2_PIN,ENKODER_PIN_1,ENKODER_PIN_2,MZ1,MZ2,ENDSTOP3_PIN_MIN,ENDSTOP4_PIN_MAKS
+from Components.Endstop import Endstop
+from Components.Motor import Motor
+from Components.Ncoder import Ncoder
+from Components.PaintSprayer import PaintSprayer
```

Ruff (0.11.9)

`Components.PaintSprayer.PaintSprayer` imported but unused

Remove unused import: `Components.PaintSprayer.PaintSprayer`

(F401)